roles/ceph-osd/tasks/pre_requisite.yml

```diff
@@ -0,0 +1,13 @@
+---
+- name: install dependencies
+  apt: pkg=parted
+       state=present
+  when: ansible_os_family == 'Debian'
```

Comment:
The assumption is that we are running on Ubuntu. No need to conditionalize here.